recipes/glpk/meta.yaml

```diff
@@ -11,10 +11,6 @@ source:
 
 build:
   number: 0
-  features:
-    - vc9      [win and py27]
```

Comment:
One way to confirm is to have a python build for the test, like in bioconda/bioconda-recipes#799